charts/private-action-runner/README.md

```diff
@@ -13,24 +22,10 @@ This Helm Chart deploys the Datadog Private Action runner inside a Kubernetes cl
 ## Use this chart
 
 1. Go to the [private action runner tab](https://app.datadoghq.com/workflow/private-action-runners).
-2. Create a new private action runner.
-3. Follow the instructions. You now have a running docker container and `config/config.yaml` file.
-4. Stop the docker container (`docker stop <name-of-the-container>` or `docker compose stop`).
-5. Create a `config.yaml` file with the appropriate values. An example `config.yaml` file is provided in the `examples` directory for you to copy.
-    * Replace the `URN_FROM_CONFIG` and the `PRIVATE_KEY_FROM_CONFIG` in the example file with with the `urn` and the `privateKey` from the `config/config.yaml` of the docker container.
-    * You can reconfigure other values or use the defaults in the example.
-6. Add this repository to your Helm repositories:
-    ```
-    helm repo add datadog https://helm.datadoghq.com
-    helm repo update
-    ```
-7. Install the Helm chart:
-    ```bash
-        helm install <RELEASE_NAME> datadog/private-action-runner -f ./config.yaml
-    ```
-8. Go to the [Workflow connections page](https://app.datadoghq.com/workflow/connections).
-9. Create a new connection, select your private action runner, and use **Service account authentication**.
-10. Create a new workflow and use a Kubernetes action like **List pod** or **List deployment**.
+2. Create a new private action runner and follow the instructions for Kubernetes.
+3. Go to the [Workflow connections page](https://app.datadoghq.com/workflow/connections).
+4. Create a new connection, select your private action runner, and use **Service account authentication**.
```

Comment:
steps 4 and 5 are specific to when a customer intends to use kubernetes actions. i'd split it out into a separate section.

ultimately i think our public docs should have action-specific instructions

Reply:
true, I split it into it's own section